index.html

```diff
@@ -0,0 +1,158 @@
+            </ul>
+          </nav>
+        </section>
+        <section class="middleSection">
+          <div class="topSection">
+            <div class="contentArea">
+              <h2>RMedia</h2>
+              <p> Lorem lpsum is simply dummy text of the printing and typesetting industry.
+              Lorem lpsum has been the industry's standard dummy text ever since the 1500's,when an
+              unknown printer took a galley of type and scrambled it to make a type specimen book.It has
+              survived not only five centuries, but also the leap into electronic typesetting,remaining
+              1960s with the release ofLetraset sheets containing lorem lpsum passages,and more recently with desktop publishing
+              software
+              like Aldus Pagemker including versions of lorem lpsum.</p>
+              <p>why do use it?</p>
+            </div>
+            <div class="imageArea">
+              <img src="resources/images/main-image.jpg" alt="Rmedia image" class="content-img">
+            </div>
+          </div>
+          <div class="bottomSection">
+            <table>
+              <thead>
+                <tr>
+                  <th>Lorem</th>
+                  <th>Lorem</th>
+                  <th>Lorem</th>
+                  <th>Lorem</th>
+                </tr>
+              </thead>
+              
+              <tbody>
+                <tr>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td><button>Apply now</button></td>
+                </tr>
+                <tr>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td><button>Apply now</button></td>
+                </tr>
+                <tr>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td><button>Apply now</button></td>
+                </tr>
+                <tr>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td><button>Apply now</button></td>
+                </tr>
+                <tr>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td><button>Apply now</button></td>
+                </tr>
+                <tr>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td><button>Apply now</button></td>
+                </tr>
+                <tr>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td>Lorem</td>
+                  <td><button>Apply now</button></td>
+                </tr>
+              </tbody>
+              
+            </table>
+          </div>
+        </section>
+        
+        <section class="rightSection col span-1-of-4">
```

Comment:
use meaningful class names. 'span-1-of-4' does not have any specific style associated with it. Remove unwanted class names

Reply:
Unnecessary class names removed.